pegasus/forms/hoc_survey_prize_2015.rb

```diff
@@ -9,9 +9,10 @@ def self.normalize(data)
     result
   end
 
-  def self.process(data)
```

Comment:
See https://github.com/code-dot-org/code-dot-org/blob/865ca031618ff16b02b8291bde2dd5776b7e48bc/bin/process_forms#L53-L57

process_ receives the entire form, while process only receives the parsed json data field. 

As to why they are named that way, I can't tell you. I'd prefer process_form & process_data or something similar but I don't think it's worth updating all the existing forms.